go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/kubevirt/kubevirt-tekton-tasks
 
-go 1.22.6
+go 1.22.7
 
 // Kubernetes
 replace (
-	k8s.io/client-go => k8s.io/client-go v0.30.3
+	k8s.io/client-go => k8s.io/client-go v0.31.0
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20240709000822-3c01b740850f
-	k8s.io/kubectl => k8s.io/kubectl v0.30.3
-	kubevirt.io/containerized-data-importer => kubevirt.io/containerized-data-importer v1.59.0
-	kubevirt.io/containerized-data-importer-api => kubevirt.io/containerized-data-importer-api v1.59.0
+	k8s.io/kubectl => k8s.io/kubectl v0.31.0
+	kubevirt.io/containerized-data-importer => kubevirt.io/containerized-data-importer v1.59.1
```

Comment:
Latest is `v1.60.3`

Reply:
These lines look the same? IIUC CDI 1.60 is the designated version for KubeVirt 1.4.

Reply:
but in CDI they are not
```
Resources corev1.VolumeResourceRequirements `json:"resources,omitempty"` - 1.60
vs
Resources corev1.ResourceRequirements `json:"resources,omitempty"` - 1.59
```
https://github.com/kubevirt/containerized-data-importer/blob/release-v1.59/staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go#L82

https://github.com/kubevirt/containerized-data-importer/blob/release-v1.60/staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go#L82